src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+import { Response } from '../../../api-objects/GenericResponse';
+import { AdminPatientResponse, ResetPasswordResponse, UserListItem } from '../../../api-objects/AdminPatient';
+import { PatientResponse } from '../../../api-objects/patientResponse';
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createPatient(patientId : string, deceased : boolean, patientNotes : string) : Observable<Response<PatientResponse>> {
+    return this.serverApiService.post('patient', {'patientId': patientId, 'deceased': deceased, 'patientNotes': patientNotes});
+  }
+
+  public getPatients() : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient');
+  }
+
+  public getPatient(patientId : number) : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient/' + patientId);
+  }
```

Comment:
This should return Appointment[]

Reply:
I'm pretty sure this should return PatientResponse. The GetPatient function is supposed to grab the Patients available on the Database. Not their related appointments.
We should have a function called getAppointment if we want an appointment array.

Reply:
Okay, then it makes even less sense to be an array.

Reply:
That's just imitating what the server is sending it (An array of 1 Patient). It's illogical, but we can fix it next week if we think it's necessary.